### PR TITLE
Added peer addr function

### PR DESCRIPTION
**Pull request by carlosb1**

### CODE REVIEW

File: src/net/udp.rs [hunk 1]
```diff
@@ -161,6 +161,33 @@ impl UdpSocket {
         self.inner.local_addr()
     }
 
+    /// Returns the socket address of the remote peer this socket was connected to.
+    ///
+    /// # Examples
+    ///
+    ///
+    #[cfg_attr(all(feature = "os-poll", not(target_os = "freebsd")), doc = "```")]
```
Comment (Thomasdezeeuw) [inline src/net/udp.rs:169]: Why are you excluding FreeBSD?

**Review by Thomasdezeeuw — APPROVED** [review]
LGTM with a single comment.

File: src/net/udp.rs (hunk 1 above)
Comment (carlosb1) [reply src/net/udp.rs:169]: I copied the description for the `addr` function, I can remove this exception if it not necessary.

Comment (Thomasdezeeuw) [reply src/net/udp.rs:169]: I guess you copied from `local_addr`? The comment above it explains why, I don't think it's needed for `peer_addr`, so you can remove it.

Comment (carlosb1) [reply src/net/udp.rs]: Yes.. I have the same doubts for other `cfg_attr` , I don-t have clear the objective for the `os-poll` feature

**Review by Thomasdezeeuw — APPROVED** [review]

**Comment by Thomasdezeeuw** [comment]
Thanks @carlosb1.

**Pull request merged**